cli/install/install.go

```diff
@@ -281,7 +281,7 @@ func isPackageInstalled(packageName string) bool {
 }
 
 // programDependenciesInstalled checks if dependencies are installed.
-func programDependenciesInstalled(program string) bool {
+func programDependenciesInstalled(program string) error {
```

Comment:
The function name + the return type became confusing. "error" should be used if something goes wrong with the dependencies checking.

Reply:
Have changed it to `(bool, error)`. So now function name corresponds to the return value and the error occurs when something goes wrong with dependencies checking and goes on higher layers of program to correctly finish a program.